install/cluster_synced.go

```diff
@@ -705,12 +705,14 @@ func (c *SyncedCluster) Ssh(sshArgs, args []string) error {
 		"-o", "StrictHostKeyChecking=no",
 	}
 	allArgs = append(allArgs, sshArgs...)
+	if c.IsLocal() {
+		allArgs = append(allArgs,
+			"-t",
```

Comment:
`-t` shouldn't be conditioned on whether it's local or not; it should be based on whether it's interactive. (or if you take my suggestion below, just remove `args` and then `roachprod ssh` is always interactive)